h5t_shim.go

```diff
@@ -261,6 +261,7 @@ var (
 
 //
 var h5t_VARIABLE int64 = C.H5T_VARIABLE
+var H5S_UNLIMITED int64 = C.H5S_UNLIMITED
```

Comment:
It needs to be exported. For instance, is passed to hdf5.CreateSimpleDataspace() to create an unlimited dataspace.